src/server/graphQLschema/type/book.type.js

```diff
@@ -64,7 +65,8 @@ const BookType = new GraphQLObjectType({
             .catch(reject);
         });
       }
-    }
+    },
+    snippets: { type: GraphQLJSON },
```

Comment:
`GraphQLList(GraphQLJSON)` makes some errors. I thought `GraphQLJSON` and `GraphQLList(GraphQLJSON)` are same but maybe they had some differences.